usr.sbin/bluetooth/iwmbtfw/iwmbtfw.8

```diff
@@ -28,8 +28,7 @@
 .Os
 .Sh NAME
 .Nm iwmbtfw
-.Nd firmware download utility for Intel Wireless 7260/8260/8265 chip based Bluetooth
-USB devices
+.Nd download firmware for Intel Wireless 7260/8260/8265 Bluetooth USB devices
```

Comment:
How do you feel about 
```suggestion
.Nd download firmware for Intel Wireless AC Bluetooth USB devices
```
? Neither one is clear. The existing cherry picks a few. The above seems better since the series is called wireless AC, but we also have another driver that does that.